### ISSUE TITLE
The file launched on the Neovim command line has not been calculated. Is there any way to include the launched file.

### ISSUE THREAD

**Issue by MaJinjie** [nvim-telescope/telescope-frecency.nvim#231]

**Comment by delphinus** [comment]
I tried this process below and I cannot reproduce the problem. @MaJinjie show me the minimal reproducible `iniv.lua`.

----

1. Prepare minimal `init.lua` into `~/.config/nvim-frecency/init.lua`. 

   <details><summary><code>init.lua</code></summary>
   
   ```lua
   local lazypath = vim.fn.stdpath "data" .. "/lazy/lazy.nvim"
   if not vim.uv.fs_stat(lazypath) then
     vim.fn.system {
       "git",
       "clone",
       "--filter=blob:none",
       "https://github.com/folke/lazy.nvim.git",
       "--branch=stable", -- latest stable release
       lazypath,
     }
   end
   vim.opt.rtp:prepend(lazypath)
   
   require("lazy").setup {
     "nvim-lua/plenary.nvim",
     "nvim-telescope/telescope.nvim",
     {
       "nvim-telescope/telescope-frecency.nvim",
       config = function()
         require("telescope").load_extension "frecency"
       end,
     },
     { "nvim-treesitter/nvim-treesitter", build = ":TSInstallSync lua" },
   }
   ```
   </details>

2. If you have tried before, remove cache.

   ```bash
   rm -fr ~/{.cache,.local/share,.local/state}/nvim-frecency/
   ```

3. Install plugins.

   ```bash
   env NVIM_APPNAME=nvim-frecency nvim +q
   ```

5. Open telescope-frecency and you see no candidates.

   ```bash
   env NVIM_APPNAME=nvim-frecency nvim +'Telescope frecency'
   ```

4. Open any file with `nvim` + filename.

   ```bash
   env NVIM_APPNAME=nvim-frecency nvim +q ~/.config/nvim/init.lua
   ```

5. Open telescope-frecency and you can see `~/.config/nvim/init.lua` has been registered into DB.

   ```bash
   env NVIM_APPNAME=nvim-frecency nvim +'Telescope frecency'
   ```

**Comment by delphinus** [comment]
I'll close this, but feel free to reopen and post a minimal reproducible `init.lua`.

**Comment by delphinus** [comment]
You might load this plugin lazily? Then you cannot register files specified after `nvim` command, such as `nvim foo.lua`, because the registering phase is done after this plugin is loaded.

I have noticed this, so made small fix: #234. I've written a note for this there and the doc.
